src/data/dtoOptional.json

```diff
@@ -45,7 +47,7 @@
   "match-v4.ParticipantStatsDto.item6": false,
 
   "match-v4.ParticipantStatsDto.visionWardsBoughtInGame": false,
-  "match-v4.ParticipantStatsDto.wardsKilled": false,

```

Comment:
Can just completely remove this line since `"match-v4.ParticipantStatsDto.*"` above handles it